reference/library/src/webapp/editor/ckeditor.launch.js

```diff
@@ -30,6 +30,21 @@ function findPos(obj) {
         return [curleft,curtop];
     }
 }
+
+//http://stackoverflow.com/a/1038781/3708872
```

Comment:
Is it worth making this only available inside the launch function as some other javascript may have a getWidth() in the global scope that may cause problems?